lib/trino-parquet/src/main/java/io/trino/parquet/ParquetTypeUtils.java

```diff
@@ -208,6 +210,18 @@ public static ColumnIO lookupColumnByName(GroupColumnIO groupColumnIO, String co
         return null;
     }
 
+    @Nullable
```

Comment:
Not pretty, but consistent with other methods in this file. A change to Optional can be a follow up, if needed.